modules/core/src/v2/coins/bsv.ts

```diff
@@ -47,31 +48,13 @@ export class Bsv extends Bch {
     return common.Environments[this.bitgo.getEnv()].bsvExplorerBaseUrl + url;
   }
 
-  getAddressInfoFromExplorer(addressBase58: string): Bluebird<any> {
-    const self = this;
-    return co(function* getAddressInfoFromExplorer() {
-      // TODO BG-9989: Update this method with the correct API route and parsing once we have one
-      const addrInfo = yield request.get(self.recoveryBlockchainExplorerUrl(`/addr/${addressBase58}`)).result();
-
-      addrInfo.txCount = addrInfo.txApperances;
-      addrInfo.totalBalance = addrInfo.balanceSat;
-
-      return addrInfo;
-    }).call(this);
+  async getAddressInfoFromExplorer(addressBase58: string, apiKey?: string): Bluebird<any> {
+    const explorer = new BlockchairApi(this.bitgo, 'bitcoin-sv', apiKey);
+    return explorer.getAccountInfo(addressBase58);
   }
 
-  getUnspentInfoFromExplorer(addressBase58: string): Bluebird<any> {
-    const self = this;
-    return co(function* getUnspentInfoFromExplorer() {
-      // TODO BG-9989: Update this method with the correct API route and parsing once we have one
-      const unspents = yield request.get(self.recoveryBlockchainExplorerUrl(`/addr/${addressBase58}/utxo`)).result();
-
-      unspents.forEach(function processUnspent(unspent) {
-        unspent.amount = unspent.satoshis;
-        unspent.n = unspent.vout;
-      });
-
-      return unspents;
-    }).call(this);
+  async getUnspentInfoFromExplorer(addressBase58: string, apiKey?: string): Bluebird<any> {
```

Comment:
Return type incorrect (getUnspents() returns a native Promise). Please wrap in `Bluebird.resolve()` to convert to a Bluebird promise.